src/db/migrations.rs

```diff
@@ -74,17 +74,14 @@ fn all_migrations() -> Vec<Box<dyn Migration>> {
       repo varchar not null,
       channel varchar not null,
       force_push_notify tinyint not null,
-      force_push_reapply_statuses varchar not null,
```

Comment:
opting for no migration path here since this change is not deployed anywhere yet. just removing for simplicity.